src/components/ppom/transactions.js

```diff
@@ -139,20 +136,18 @@ export function ppomMaliciousTransactionsAndSignatures(parentContainer) {
 
   document.addEventListener('newNetwork', function (e) {
     mintSepoliaERC20.hidden = !isSepoliaNetworkId(e.detail.networkId);
-    maliciousContractInteractionButton.hidden =
-      isBaseNetworkId(e.detail.networkId) ||
-      isSepoliaNetworkId(e.detail.networkId);
```

Comment:
this seems a mistake, as the logic prevented to show the Contract Interaction button for Sepolia (unsure why this was added). Now we can see the button back again, and test this flow